lockbox-ios/Presenter/RootPresenter.swift

```diff
@@ -217,7 +225,15 @@ class RootPresenter {
 }
 
 extension RootPresenter {
-    private func unlockBlindly() {
+    fileprivate func startTelemetry() {
+        Observable.combineLatest(self.telemetryStore.telemetryFilter, self.userDefaults.onRecordUsageData)
+                .filter { $0.1 }
+                .map { $0.0 }
+                .bind(to: self.telemetryActionHandler.telemetryActionListener)
+                .disposed(by: self.disposeBag)
+    }
+
+    fileprivate func unlockBlindly() {
```

Comment:
Nit: naming/documentation. What does this do?

Reply:
This is a holdover method that keeps the Javascript datastore "unlocked" (i.e.) unenecrypted when the app is open.